### DISCUSSION TITLE
tui-tabs изменить цвет подчеркивания

### DISCUSSION THREAD

**Discussion by expdts** [taiga-family/taiga-ui#5092] [Q&A] [answered]
Изменил цвет подчеркивания вкладки, но при переходе к другой вкладки появляется цвет по умолчанию. Подскажите пожалуйста, как я могу изменить на свой цвет для подчеркивания?
```
tui-tabs {
  &[data-vertical] {
    ._active {
      &:after {
        background: red !important;
      }
    }
  }
}
```
![image](https://github.com/taiga-family/taiga-ui/assets/109337333/886ba6d8-5d97-487a-b1ad-b96caf31ee33)

**Comment by waterplea** [comment]
Ты можешь переопределить `--tui-primary` CSS переменную на `tui-tabs`

**Reply by expdts** [reply]
Что вы имеете ввиду? В background я как раз это сделал, но это не помогает.

**Reply by waterplea** [reply, marked as answer]
```css
tui-tabs {
  --tui-primary: red;
}
```
